src/cmd/api/controllers/video_transformer_list_handler_test.go

```diff
@@ -0,0 +1,91 @@
+package controllers_test
+
+import (
+	"net/http/httptest"
+	"sync"
+	"testing"
+
+	"github.com/Sogilis/Voogle/src/pkg/clients"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/Sogilis/Voogle/src/cmd/api/config"
+	"github.com/Sogilis/Voogle/src/cmd/api/models"
+	"github.com/Sogilis/Voogle/src/cmd/api/router"
+)
+
+var _ clients.ServiceDiscovery = &dummyServiceDiscovery{}
+
+type dummyServiceDiscovery struct {
+	transformersAddressesList map[string]*clients.TransformersInstances
+	mutex                     sync.RWMutex
+}
```

Comment:
We should create a Service_discovery_dummy.go clients as we did for other clients no?